pages/store/[nftClassId]/index.vue

```diff
@@ -487,7 +523,10 @@ const isPricingItemsVisible = useElementVisibility(pricingItemsElement)
 const pricingItems = computed(() => {
   return bookInfo.pricingItems.value.map((item, index) => ({
     ...item,
-    formattedPrice: formatPrice(item.price),
+    originalPrice: formatPrice(item.price),
+    discountedPrice: isLikerPlus.value && item.price > 0
```

Comment:
better to simplify to just check `isLikerPlus.value ` and check `item.price > 0` in sth like `getPlusDiscountPrice()`